code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -23,6 +23,11 @@
 	name = "[initial(name)] ([storedorgan.name])"
 
 /obj/item/autosurgeon/attack_self(mob/user)//when the object it used...
+	if(istype(storedorgan, /obj/item/organ/regenerative_core))
+		var/obj/item/organ/regenerative_core/cores = storedorgan
+		to_chat(user, span_notice("[src] fucking explodes!"))
+		qdel(cores)
+		explosion(get_turf(src), 0, 0, 2, flame_range = 5)
```

Comment:
Contrary to popular belief, this won't destroy the autosurgeon (and shouldnt that would be stupid). The explosion will however cause damage to armor and items the user is carrying as explosions do

Reply:
It will damage it the first time, however as long as they don't repeatedly spam it nothing will be destroyed, ill double check lavaland but that shouldn't instantly destroy anything. If it does I'll provide an update and lower the explosion.

Reply:
Checked the damage on both lavaland and the station, nothing explodes instantly. I removed the fire because that actually could burn off your clothes if you were unlucky enough.

Reply:
Sorry actually the only thing that gets destroyed is the station map.